xarray/tests/test_plot.py

```diff
@@ -2674,7 +2674,28 @@ def test_datetime_units(self):
         # test that matplotlib-native datetime works:
         fig, ax = plt.subplots()
         ax.plot(self.darray["time"], self.darray)
-        assert isinstance(ax.xaxis.get_major_locator(), mpl.dates.AutoDateLocator)
+        assert type(ax.xaxis.get_major_locator()) is mpl.dates.AutoDateLocator
+
+    def test_datetime_plot1d(self):
+        # Test that matplotlib-native datetime works:
+        p = self.darray.plot.line()
+        ax = p[0].axes
+        assert type(ax.xaxis.get_major_locator()) is mpl.dates.AutoDateLocator
```

Comment:
Using `type` here instead of `isinstance` because pandas inherits from `mpl.dates.AutoDateLocator`:
https://github.com/pandas-dev/pandas/blob/140b1c2975437f4790dcf62cd49eac8ce8896a5e/pandas/plotting/_matplotlib/converter.py#L346

Reply:
Add this as a comment?

Reply:
Done.